tokio-util/CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
-# 0.3.0 (February 28, 2020)
+# 0.3.0 (March 4, 2020)
 
-Breaking changes:
+### Changed
 
-- Change codec::Encoder trait to take a generic Item parameter (#1746), which allows
-  codec writers to pass references into `Framed` and `FramedWrite` types.
+- Change `Encoder` trait to take a generic `Item` parameter, which allows
```

Comment:
we should probably make it clear that this is a breaking change in the changelog...
```suggestion
- **Breaking Change**: Change `Encoder` trait to take a generic `Item` parameter, which allows
```